include/PathOrderOptimizer.h

```diff
@@ -180,6 +182,23 @@ class PathOrderOptimizer
             }
         }
 
+        // Set actual used start point calculation strategy for each path
+        const OrderablePath* previous_path = nullptr;
+        for (auto& path : paths_)
+        {
+            if (use_shortest_for_inner_walls_ && previous_path && previous_path->is_outer_wall && ! path.is_outer_wall)
```

Comment:
Well I think we don't really care about starting position for anything else than outer walls, but that is a good question :thinking: I also remember we have a ticket to set the start position of infill to shortest distance, so that may actually tend to it. Let's PPM give an opinion on this.